File: src/Python/bubblesort.py
def bubbleSort(lista):
    for i in range(len(lista)): #percorre a lista
        for j in range(len(lista)-1): #percorre a lista -1
            if lista[j] > lista[j+1]: #se o elemento da posição j for maior que o elemento da posição j+1
                lista[j],lista[j+1] = lista[j+1],lista[j] #troca os elementos de posição
    return lista #retorna a lista ordenada

lista = [5,4,3,2,1]
print(bubbleSort(lista))

File: src/JavaScript/bubblesort.js
function bubbleSort(array) {
    let verificarTroca = true; // variável que verifica se houve troca
    while (verificarTroca) {
        verificarTroca = false; // se não houver troca, o loop para
        for (let i = 0; i < array.length; i++) {
        if (array[i + 1] < array[i]) { // se o próximo elemento for menor que o atual
            [array[i], array[i + 1]] = [array[i + 1], array[i]];   // troca os elementos de lugar
            verificarTroca = true; // se houve troca, o loop continua
        }
        }
    }
    return array; // retorna o array ordenado
    }

    console.log(bubbleSort([5, 4, 3, 2, 1])); // [1, 2, 3, 4, 5]